lms/djangoapps/verify_student/services.py

```diff
@@ -0,0 +1,22 @@
+from opaque_keys.edx.keys import CourseKey
```

Comment:
Please add docstrings for this module, the `ReverificationService` class, and the methods.